lib/api/core/models/repositories/profileRepository.js

```diff
@@ -115,6 +117,19 @@ class ProfileRepository extends Repository {
     return Bluebird.all(profiles);
   }
 
+  /**
+   * @override
+   */
+  loadOneFromDatabase (id) {
```

Comment:
Without this override, the error returned in case of a 404 would be a generic `services.storage.not_found`